File: Latihan1b/C++/TimSepakBola.cpp
#include <string>
#include <iostream>

using namespace std;

class TimSepakBola
{
    private:
        string nama;
        string asalNegara;
        string tahunBerdiri;
        string pemain;
    
    public:
        // constructor
        TimSepakBola() {}

        TimSepakBola(string nama, string asalNegara, string tahunBerdiri, string pemain)
        {
            this->nama = nama;
            this->asalNegara = asalNegara;
            this->tahunBerdiri = tahunBerdiri;
            this->pemain = pemain;
        }

        // setter and getter method
        void setNama(string nama)
        {
            this->nama = nama;
        }

        string getNama()
        {
            return this->nama;
        }

        void setAsalNegara(string asalNegara)
        {
            this->asalNegara = asalNegara;
        }

        string getAsalNegara()
        {
            return this->asalNegara;
        }

        void setTahunBerdiri(string tahunBerdiri)
        {
            this->tahunBerdiri = tahunBerdiri;
        }

        string getTahunBerdiri()
        {
            return this->tahunBerdiri;
        }

        void setPemain(string pemain)
        {
            this->pemain = pemain;
        }

        string getPemain()
        {
            return this->pemain;
        }

        // output
        void printOut()
        {
            cout << "Nama Tim       : " << this->getNama() << endl;
            cout << "Asal Negara    : " << this->getAsalNegara() << endl;
            cout << "Tahun Berdiri  : " << this->getTahunBerdiri() << endl;
            cout << "Pemain         : " << this->getPemain() << endl;
        }

        ~TimSepakBola() {}
};

File: Latihan1b/PHP/TimSepakBola.php
<?php

class TimSepakBola
{
    private $nama = null;
    private $asalNegara = null;
    private $tahunBerdiri = null;
    private $pemain = null;

    // constructor
    public function __construct($nama = null, $asalNegara = null, $tahunBerdiri = null, $pemain = null)
    {
        $this->nama = $nama;
        $this->asalNegara = $asalNegara;
        $this->tahunBerdiri = $tahunBerdiri;
        $this->pemain = $pemain;
    }

    // setter and getter method
    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function setAsalNegara($asalNegara)
    {
        $this->asalNegara = $asalNegara;
    }

    public function getAsalNegara()
    {
        return $this->asalNegara;
    }

    public function setTahunBerdiri($tahunBerdiri)
    {
        $this->tahunBerdiri = $tahunBerdiri;
    }

    public function getTahunBerdiri()
    {
        return $this->tahunBerdiri;
    }

    public function setPemain($pemain)
    {
        $this->pemain = $pemain;
    }

    public function getPemain()
    {
        return $this->pemain;
    }

    // output
    public function printOut()
    {
        echo "Nama Tim      : ". $this->getNama(). "<br/>";
        echo "Asal Negara   : ". $this->getAsalNegara(). "<br/>";
        echo "Tahun Berdiri : ". $this->getTahunBerdiri(). "<br/>";
        echo "Pemain        : ". $this->getPemain(). "<br/>";
    }
}

?>